kernel-ewasm/tests/syscalls/entry.ts

```diff
@@ -0,0 +1,73 @@
+const Web3 = require('web3')
+const assert = require('assert')
+const fs = require('fs')
+
+import { newKernelInstance, web3, createAccount, KernelInstance, deployContract, normalize, EntryCap, WriteCap, RegisterCap, NewCap} from '../utils'
+import { Tester, TestContract } from '../utils/tester';
+import { notEqual } from 'assert';
+
+
+describe('Set Entry Syscall', function () {
+    this.timeout(40_000);
+    describe('set entry', function () {
+        it('set the entry to writer, execute writer', async function () {
+            const tester = new Tester();
```

Comment:
Is the `Tester` module planned to be introduced in all other tests? It's just that all the other tests are quite readable on their own at the current time.

Reply:
`procedure_call.ts` for example? I find `log.ts` much easier to read and manage. The difference here is that I'm combining boilerplate from various tests into a single file. This is because (for example) the delete, and set entry tests also require register as well.

Reply:
Ok sounds good!